src/tabs/PullRequestsTab.tsx

```diff
@@ -549,6 +517,34 @@ export class PullRequestsTab extends React.Component<
     this.reloadPullRequestItemProvider(filteredPullRequest);
   }
 
+  private getFilterData<T>(key: string): T | undefined {
+    let filterData = this.filter.getFilterItemValue<T>(key);
+
+    if (filterData === undefined) {
+      const cachedData = this.getFilterFromCache<T>(key);
+      if (cachedData !== null) {
+        filterData = cachedData;
+      }
+    } else {
+      this.storeFilterData(key, filterData);
+    }
+
+    return ;
+  }
+
+  private storeFilterData(key: string, data: any): void {
+    localStorage.setItem(`${STORED_CACHE_KEY_PREFIX}${key}`, JSON.stringify(data));
```

Comment:
What about introducing `const` for this value?

Reply:
I'm not sure if I got your point. Which value do you refer?